spark-stream-ingest/src/main/python/requirements.txt

```
@@ -0,0 +1,4 @@
+pandas==1.2.4
+cortex-python==6.1.0
+delta-spark==1.1.0
```

Is there a problem using the newest version (1.2.1)
```suggestion
delta-spark==1.1.0
```

---

Delta 1.2.0 was just released in April, probably when we were doing the 1.1.0 upgrade. This is not a component that we can just update without a significant amount of testing. It would need to be as part of a release. If it has to be done it will need to be a ticket. Without a vuln tied to it or some feature we need from it, I don't want to upgrade this at this time.